stockfish/models.py

```diff
@@ -147,6 +147,10 @@ def _put(self, command: str) -> None:
     def _read_line(self) -> str:
         if not self._stockfish.stdout:
             raise BrokenPipeError()
+        if self._stockfish.poll() is not None:
+            raise ValueError(
```

Comment:
Let's create custom exception. For example `StockfishException` with text without details about `invalid FEN`. Make it more universal, please.